packages/create-yoshi-app/templates/business-manager-module/javascript/dev/environment.js

```diff
@@ -1,19 +1,21 @@
+import { readFileSync } from 'fs';
 import {
   createTestkit,
   testkitConfigBuilder,
-  ModuleConfigFileEmitter,
+  anAppConfigBuilder,
 } from '@wix/business-manager/dist/testkit';
 
 const getTestKitConfig = async (
   { withRandomPorts } = { withRandomPorts: false },
 ) => {
   const serverUrl = 'http://localhost:3200/';
-  const path = './templates/module_{%PROJECT_NAME%}.json.erb';
+  const path = './templates/module_{%PROJECT_NAME%}.json';
   const serviceId = 'com.wixpress.{%projectName%}';
 
-  const moduleConfig = await new ModuleConfigFileEmitter(path)
-    .registerStaticService({ serviceId, serverUrl })
-    .emit();
+  const moduleConfig = anAppConfigBuilder()
+    .fromJsonTemplate(JSON.parse(readFileSync(path).toString()))
```

Comment:
Consider simply:
```suggestion
    .fromJsonTemplate(require(path))
```

Reply:
We should also add the correct way to do it in a comment or something:
```suggestion
	.fromJsonTemplate(require(path)) // For dev
    // .fromModuleId('{%PROJECT_NAME%}') // For production value
```